cookbooks/aws-parallelcluster-platform/spec/unit/resources/nvidia_repo_spec.rb

```diff
@@ -127,7 +127,7 @@ def self.remove(chef_run)
         when 'amazon', 'centos'
           'rhel7'
         when 'redhat'
-          'rhel8'
+          "rhel#{version}"
```

Comment:
Why in thre change above it is required a `.to_i` and here it is now?

Reply:
For Ubuntu `.to_i` is required because we want to convert `22.04` to `22`. For RHEL `.to_i` is optional